dev/benchmarks/macrobenchmarks/lib/web_benchmarks.dart

```diff
@@ -310,23 +338,36 @@ class TimeseriesVisualization {
 /// implement a manual fallback. This allows debugging benchmarks using plain
 /// `flutter run`.
 class LocalBenchmarkServerClient {
+  LocalBenchmarkServerClient(this.serverOrigin);
+
   /// This value is returned by [requestNextBenchmark].
   static const String kManualFallback = '__manual_fallback__';
 
+  /// The origin (e.g., http://localhost:1234) of the benchmark server that
+  /// hosts the macrobenchmarking API.
+  final Uri serverOrigin;
+
   /// Whether we fell back to manual mode.
   ///
   /// This happens when you run benchmarks using plain `flutter run` rather than
   /// devicelab test harness. The test harness spins up a special server that
   /// provides API for automatically picking the next benchmark to run.
   bool isInManualMode = false;
 
+  Map<String, String> get headers => <String, String>{
```

Comment:
```suggestion
  Map<String, String> get headers => const <String, String>{
```

?

Reply:
`serverOrigin` isn't const, unfortunately. We ideally want this to be passed in via clarg - but that's not supported for browser unfortunately.

Reply:
Gah! I can read. sorry.